packages/utils/src/route.js

```diff
@@ -4,24 +4,37 @@ import consola from 'consola'
 
 import { r } from './resolve'
 
+const getRouteChildrens = function (route) {
+  if (route.children.some(child => child.path === '')) {
```

Comment:
nit: extracting the condition as variable and name it, e.g. 
`const routeHasChildWithEmptyPath = route.children.some(child => child.path === '')`

Probably something less verbose though